website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -48,17 +47,17 @@
 
 Use color logically.
 
-- **Nuetral** to provide general information to the user about an ongoing process.
-- **Highlight** use interchangeably with neutral when more prominence is needed. Use it sparingly.
-- **Success** to inform users that an action has been completed successfully.
-- **Warning** to indicate that an action was executed successfully but may have triggered a related issue. Provide guidance and actions if possible.
-- **Critical** to indicate error or critical issues as a result of a failed action.
+- **Neutral** to provide general information to the user about an ongoing process.
+- **Highlight** use interchangeably with `neutral` when more prominence is needed. Use it sparingly.
+- **Success** to indicate a successful action was completed.
+- **Warning** to indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
+- **Critical** to indicate error or critical issues resulting from a failed action.
 
----
+## Icons
 
-### Icons
+All Toasts have icons by default that are intentionally tied to the Toast type.
```

Comment:
Should this be "Toast color" rather than type?

Reply:
Good catch! Will fix